spec/features/user_sign_out_spec.rb

```diff
@@ -0,0 +1,22 @@
+feature 'User signs out' do
+  let(:user) do
+    User.create(username: 'Teeohbee',
+                password: '12345678',
+                email: 'toby@example.com',
+                password_confirmation: '12345678')
+  end
+
+  scenario 'while being signed in' do
+    sign_in(email: 'toby@example.com', password: '12345678')
+    click_button 'Sign out'
+    expect(page).to have_content('You successfully signed out')
+    expect(page).not_to have_content('Welcome, Teeohbee')
+  end
+
+  def sign_in(email:, password:)
```

Comment:
Unused method argument - `email`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `password`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.